openmetadata-ui/src/main/resources/ui/src/components/UserList/UserListV1.tsx

```diff
@@ -0,0 +1,265 @@
+                <Button
+                  icon={
+                    <SVGIcons
+                      alt="Restore"
+                      className="tw-w-4"
+                      icon={Icons.RESTORE}
+                    />
+                  }
+                  type="text"
+                  onClick={() => {
+                    setSelectedUser(record);
+                    setShowReactiveModal(true);
+                  }}
+                />
+              </Tooltip>
+            )}
+            <Tooltip placement="bottom" title="Delete">
+              <Button
+                icon={
+                  <SVGIcons
+                    alt="Delete"
+                    className="tw-w-4"
+                    icon={Icons.DELETE}
+                  />
+                }
+                type="text"
+                onClick={() => {
+                  setSelectedUser(record);
+                  setShowDeleteModal(true);
+                }}
+              />
+            </Tooltip>
+          </Space>
+        ),
+      },
+    ];
+  }, [showRestore]);
+
+  return (
+    <Row className="user-listing" gutter={[16, 16]}>
+      <Col span={8}>
+        <Searchbar
+          removeMargin
+          placeholder="Search for user..."
+          searchValue={searchTerm}
+          typingInterval={500}
+          onSearch={onSearch}
+        />
+      </Col>
+      <Col span={16}>
+        <Space align="center" className="tw-w-full tw-justify-end" size={16}>
+          <span>
+            <Switch
+              checked={showDeletedUser}
+              onClick={onShowDeletedUserChange}
+            />
+            <span className="tw-ml-2">Deleted Users</span>
+          </span>
+          <Button type="primary" onClick={handleAddNewUser}>
+            Add User
+          </Button>
+        </Space>
+      </Col>
+
+      <Col span={24}>
+        <Table
+          className="user-list-table"
+          columns={columns}
+          dataSource={data}
+          loading={{
+            spinning: isDataLoading,
+            indicator: <Loader size="small" />,
+          }}
+          pagination={false}
+          size="middle"
+        />
+      </Col>
+      <Col span={24}>
+        {paging.total > PAGE_SIZE && (
+          <NextPrevious
+            currentPage={currentPage}
+            isNumberBased={Boolean(searchTerm)}
+            pageSize={PAGE_SIZE}
+            paging={paging}
+            pagingHandler={onPagingChange}
+            totalCount={paging.total}
+          />
+        )}
+      </Col>
+
+      <Modal
+        cancelButtonProps={{
+          type: 'link',
+        }}
+        className="reactive-modal"
+        okText="Reactive"
+        title="Reactive User"
+        visible={showReactiveModal}
+        onCancel={() => {
+          setShowReactiveModal(false);
+          setSelectedUser(undefined);
+        }}
+        onOk={handleReactiveUser}>
+        <p>Are you sure you want to reactive {getEntityName(selectedUser)}?</p>
```

Comment:
Can we use "Restore" instore of reactivate?
```suggestion
        <p>Are you sure you want to restore User {getEntityName(selectedUser)}?</p>
```

Reply:
Yes, nice! ill update the same Thanks